build.gradle

```diff
@@ -16,9 +16,6 @@ plugins {
 
 subprojects {
   apply plugin: "org.jmailen.kotlinter" // Version should be inherited from parent
-  kotlinter {
```

Comment:
Upgrade kotlinter in L13 to `3.14.0`, and this should pass.

Reply:
Weird that this worked for me locally. Perhaps I didn't run `./gradlew lintKotlin` after removing the `kotlinter`-config.

Anyway, if I upgrade to either `3.14.0` or `3.15.0` we get other rule violations, as those versions enforce additional rules. All of those rules are automatically formattable, but as per the last PR, I'll leave that formatting commit to you.

You can create it by calling `./gradlew formatKotlin` - it mostly adds trailing commas to basically everything.

Reply:
K thx, I'll do that shortly.